plaidml/op/lib/ops.cc

```diff
@@ -2046,8 +2047,8 @@ Tensor compute_iou(std::vector<Tensor> Boxes_coordinates, TensorDim num_batches,
 Value nms(const Value& value) {
   IVLOG(1, "nms");
   auto args = value.as_tuple();
-  if (args.size() != 18) {
-    throw std::runtime_error("nms expects 18 arguments");
+  if (args.size() != 19) {
```

Comment:
Just a note: This will have merge conflict with Zhibin's PR https://github.com/Flex-plaidml-team/plaidml/pull/104